### PR TITLE
Updated build pipeline to push to E4K feed

### PR DESCRIPTION
**Pull request by tmahmood-microsoft**

### CODE REVIEW

File: vsts/build-test-nuget.yml
Comment (tmahmood-microsoft) [inline vsts/build-test-nuget.yml:1]: This file needs attention from reviewers

File: vsts/build-test-nuget.yml
Comment (patilsnr) [inline vsts/build-test-nuget.yml:459]: Have we tested this operation to see how long it takes on average? I don't have a gauge on how relative 15 minutes takes in relation to uploading the packages.

**Review by patilsnr — APPROVED** [review]
LGTM, could be worth waiting for another review to double check the yml file.

File: vsts/build-test-nuget.yml
Comment (timtay-microsoft) [inline vsts/build-test-nuget.yml:502]: Should we get a more dedicated "SDK-wide usable" service connection instead of a personal one of yours?

File: vsts/build-test-nuget.yml
Comment (tmahmood-microsoft) [reply vsts/build-test-nuget.yml:459]: on average it takes ~2-3 minutes. This timeout value is similar to the job where we upload to partner signing and release pipeline

File: vsts/build-test-nuget.yml
Comment (tmahmood-microsoft) [reply vsts/build-test-nuget.yml]: updated

File: vsts/build-test-nuget.yml
Comment (timtay-microsoft) [reply vsts/build-test-nuget.yml:459]: The timeout value doesn't need to be too restrictive here. 15 minutes is probably fine.

**Pull request merged**
